utils/fixed_decimal/src/decimal.rs

```diff
@@ -112,8 +112,17 @@ pub struct FixedDecimal {
     /// - <= magnitude
     lower_magnitude: i16,
 
-    /// Whether the number is negative. Negative zero is supported.
-    is_negative: bool,
+    // The sign; note that a positive value may be represented by either
+    // `Sign::Positive` (corresponding to a prefix +) or `Sign::None`
+    // (corresponding to the absence of a prefix sign).
+    sign: Sign,
+}
+
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
+pub enum Sign {
+    None,
+    Negative,
+    Positive,
 }
```

Comment:
Nit: This should have docs, which you can add with `///` comments. Please add docs on the enum as well as on the three variants. All exported symbols should have docs.